setup.py

```diff
@@ -461,6 +461,21 @@ def create_or_sync_database(configure):
     else:
         call(u'Migrate DB:', [u'env/bin/python', u'manage.py', u'migrate'])
 
+def load_redirects(configure):
+    from django.contrib.redirects.models import Redirect
+    from django.contrib.sites.models import Site
+
+    Redirect.objects.all().delete()
+    site = Site.objects.get(pk=1)
```

Comment:
@viliambalaz Nie je dobre brat objekty takto cez `pk`. DB ID nie je nieco, co by malo byt natvrdo v kode. Kod by mal fungovat aj ak sa v DB vygeneruju ine ID. Na identifikaciu `site` objektu pouzivaj jeho `name. Rovnako ako mame o funkciu nizsie v `configure_site_domain`.

Reply:
:ok: Vychadzal som z toho ze vo `fixtures/sites_site.json` je definovany `pk=1`.

Reply:
@viliambalaz Chapem. Aj na tom druhom projekte mame tam explicitne `pk=1`. Ale je to zle. Explicitne IDcka netreba nikde pisat. Ak uz sa inac neda, tak explicitne IDcka by mali byt v nejakych globalnych konstantach, nie takto v kode.

Pri `site` nastastie vieme pouzit `name` na identifikaciu.